### PR TITLE
Final minor edits

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This is the last set of minor edits -- removing end of line spaces, cleaning up abbreviations, etc.
The only thing is, I did suggest deleting the conclusion because I think the sections @RLordan and @aimundo added are a stronger ending!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -247,14 +247,14 @@ This makes this approach a truly portable assay for point-of-care diagnostics.
 The authors achieved detection of 100 copies/&mu;l of pre-isolated RNA in 30 minutes, and correctly identified all SARS-CoV-2-positive patient RNA samples tested in 5 minutes (n=20).
 
 There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
-Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) has also been instrumental in scaling up testing protocols.
+Technology based on CRISPR has also been instrumental in scaling up testing protocols.
```
Comment (RLordan) [inline content/10.diagnostics.md:250]: Does one of these need to be deleted? Repeated below

**Review by RLordan — APPROVED** [review]
Hey @rando2 I think there may be one repeated line. I also agree that dropping the conclusions actually ends the paper on a stronger note. Thanks!

File: content/10.diagnostics.md (hunk 1 above)
Comment (rando2) [reply content/10.diagnostics.md:250]: yes! weird, I think this one is out of place!

**Comment by agitter** [comment]
We may want to double-check the template variables.  I noticed these warnings in the build log:
```
## WARNING
Template variable warning: 'owid_vaccine_types' is undefined
## WARNING
Template variable warning: 'owid_inactivated_whole_virus_count' is undefined
## WARNING
Template variable warning: 'owid_inactivated_whole_virus_countries' is undefined
```

**Comment by rando2** [comment]
Thank you @agitter! I'll work on that now -- those are in a different manuscript so I will make a new branch to address it!

**Comment by agitter** [comment]
> those are in a different manuscript

Oh, right.  I was confused by the work on parallel manuscripts.  We can clean this up later then.  I saw another bad figure reference in the build log too that I can explore.

**Pull request merged**